### ISSUE TITLE
Sound ?

### ISSUE THREAD

**Issue by pawelwiejkut** [telmomarques/xiaomi-360-1080p-hacks#50]
Hello Guys,

Sound is not supported or I have some issues?

**Comment by KhArtNJava** [comment]
Yeah, good question. I plan to flash my cam and I didn't find info about sound in custom firmware/RTSP.
Does this firmware supports sounds.

**Comment by telmomarques** [comment]
Hi,

Sound not supported, some progress has been made but it's not stable at this point.

If you want to know more or help out check the discord badge on the readme.
